src/components/field.jsx

```diff
@@ -25,9 +25,17 @@ function fieldCounts(dimInfo, field) {
   return str;
 }
 
+function firstFewValues(layout) {
+  let result = 'Example values:\n';
```

Comment:
initiate result as empty string? Or is "Example values" supposed to be displayed somtime?

Reply:
Ah.. used in the mouseover-thingie. Nice! :)